src/lib/CrsfProtocol/crsf_protocol.h

```diff
@@ -27,6 +27,8 @@
 #define CRSF_CHANNEL_VALUE_MID  992
 #define CRSF_CHANNEL_VALUE_2000 1792
 #define CRSF_CHANNEL_VALUE_MAX  1811 // 2012us - actual CRSF max is 1984 with E.Limits on
+
+#define CRSF_MIN_PACKET_LEN 5
```

Comment:
I think min length should be 4, for correctness. For broadcast frames that _may_ have zero payload length. Not that there are any.